src/app/components/action-card-component/action-card-component.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input } from '@angular/core';
+import { ActionCardContentInterface } from './action-card-content-interface';
+
+@Component({
+  selector: 'app-action-card-component',
+  templateUrl: './action-card-component.component.html',
+  styleUrls: ['./action-card-component.component.scss'],
+})
+export class ActionCardComponentComponent {
+  isListOpen: boolean;
+  showPopover: boolean;
+  @Input() isDeletableAction!: boolean;
+  @Input() title!: string;
+  @Input() titleColor!: string;
+  @Input() addActivityText!: string;
+  @Input() deleteText!: string;
+  @Input() cardContentRow: any;
```

Comment:
do not use 'any' in your code.
it should be sth like this:
@Input() cardContentRow: ActionCardContentInterface[] = []